OpenSimRT/Common/include/CircularBuffer.h

```diff
@@ -75,15 +157,21 @@ template <int history, typename T> class CircularBuffer {
         bufferNotEmpty.notify_one();
     }
 
+    /**
+     * Retrieve data from buffer. To retrieve data, the buffer must have at
+     * least M elements, else the consumer thread is blocked until buffer is
+     * filled with M values.
```

Comment:
Point out that the blocking behavior is changed when the model = continuous, thus the get call will always return the latest M elements regardless of whether we have a new entry.